wolfcrypt/src/port/autosar/cryif.c

```diff
@@ -0,0 +1,96 @@
+/* cryif.c
+ *
+ * Copyright (C) 2006-2019 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+/* AutoSAR 4.4 */
+/* shim layer for use of wolfSSL crypto driver */
+
+
+#ifdef HAVE_CONFIG_H
+    #include <config.h>
+#endif
+
+#include <wolfssl/wolfcrypt/settings.h>
+#include <wolfssl/wolfcrypt/port/autosar/Csm.h>
+#include <wolfssl/wolfcrypt/port/autosar/CryIf.h>
+#include <wolfssl/wolfcrypt/port/autosar/Crypto.h>
+
+#ifdef WOLFSSL_AUTOSAR
+#ifndef NO_WOLFSSL_AUTOSAR_CRYIF
+
+#include <wolfssl/wolfcrypt/logging.h>
+
+/* initialization function */
+void CryIf_Init(const CryIf_ConfigType* in)
+{
+    (void)in;
+    Crypto_Init(NULL);
+}
+
+
+void CryIf_GetVersionInfo(Std_VersionInfoType* ver)
+{
+    (void)ver;
```

Comment:
Not returning anything here through ver?

Reply:
we don't have an autosar vendor/model ID but set the major/minor/patch versions now